Arrays/703KthLargestElementinaStream.py

```
@@ -0,0 +1,24 @@
+import heapq
+class Solution:
+      def __init__(self,k:int,nums:list[int]):
```

Pylint (3.3.7)

[warning] Bad indentation. Found 6 spaces, expected 4

(W0311)